vw/src/main/scala/com/microsoft/azure/synapse/ml/vw/BanditEstimatorCressieRead.scala

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.vw
+
+import org.apache.spark.sql.{Encoder, Encoders}
+import org.apache.spark.sql.expressions.Aggregator
+
+/**
+  * Cressie-Read Bandit estimator
+  *
+  * Background http://www.machinedlearnings.com/2020/12/distributionally-robust-contextual.html
+  *
+  * Python: https://github.com/VowpalWabbit/estimators/blob/03c8ba619d68f54849d4fa2da2b1a148e6cdb990/estimators/bandits/cressieread.py#L25
+  */
+class BanditEstimatorCressieRead
+  extends Aggregator[BanditEstimatorCressieReadInput, BanditEstimatorCressieReadBuffer, Double]
+  with Serializable {
+  // TODO: doesn't work
+  //    with BasicLogging {
+  //  logClass()
```

Comment:
Is this class just for transformers? How do you know it doesent work?

Reply:
fixed.